config/protractor/protractor-dev.conf.js

```diff
@@ -31,8 +31,7 @@ let config = {
           url,
           common.tmp
         ]))
-        .then(() => common.exec(`npm`, [`i`, '--only=prod'], common.cwdOpts))
-        .then(() => common.exec(`npm`, [`i`, `../`], common.cwdOpts))
```

Comment:
Gotcha. I was confused by that, since it worked when I removed it. I'll change it back and debug what's going on.